test/ExpressiveInstallerTest/InstallerTestCase.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace ExpressiveInstallerTest;
+
+use Composer\Factory;
+use Composer\IO\IOInterface;
+use Composer\Json\JsonFile;
+use ExpressiveInstaller\OptionalPackages;
+use Interop\Container\ContainerInterface;
+use Prophecy\Argument;
+use Psr\Http\Message\ResponseInterface;
+use Zend\Diactoros\Response;
+use Zend\Diactoros\ServerRequest;
+use Zend\Expressive\Application;
+
+class InstallerTestCase extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var IOInterface
+     */
+    private $io;
+
+    private $projectRoot;
+
+    /**
+     * @var ContainerInterface
+     */
+    protected $container;
+
+    protected $response;
+
+    protected $teardownFiles = [];
+
+    public function setup()
+    {
+        $this->response = null;
+
+        $this->cleanup();
+
+        $this->io = $this->prophesize('Composer\IO\IOInterface')->reveal();
```

Comment:
Shouldn't we wait to reveal this until later, so we can do assertions on what was written?